packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -381,7 +381,90 @@ const LivechatDepartmentSchema = {
 	additionalProperties: false,
 };
 
-export const isLivechatDepartmentProps = ajv.compile<LivechatDepartmentProps>(LivechatDepartmentSchema);
+export const isGETLivechatDepartmentProps = ajv.compile<LivechatDepartmentProps>(LivechatDepartmentSchema);
+
+type POSTLivechatDepartmentProps = {
+	department: {
+		enabled: boolean;
+		name: string;
+		email: string;
+		description?: string;
+		showOnRegistration: boolean;
+		showOnOfflineForm: boolean;
+		requestTagsBeforeClosingChat?: boolean;
+		chatClosingTags?: string[];
+		fallbackForwardDepartment?: string;
+	};
+	agents: {
+		upsert: string[];
+		remove: string[];
+	};
+};
+
+const POSTLivechatDepartmentSchema = {
+	type: 'object',
+	properties: {
+		department: {
+			type: 'object',
+			properties: {
+				enabled: {
+					type: 'boolean',
+				},
+				name: {
+					type: 'string',
+				},
+				description: {
+					type: 'string',
+					nullable: true,
+				},
+				showOnRegistration: {
+					type: 'boolean',
+				},
+				showOnOfflineForm: {
+					type: 'boolean',
+				},
+				requestTagsBeforeClosingChat: {
+					type: 'boolean',
+					nullable: true,
+				},
+				chatClosingTags: {
+					type: 'array',
+					items: {
+						type: 'string',
+					},
+					nullable: true,
+				},
+				fallbackForwardDepartment: {
+					type: 'string',
+					nullable: true,
+				},
+				email: {
+					type: 'string',
+				},
+			},
+			required: ['name', 'email', 'enabled', 'showOnRegistration', 'showOnOfflineForm'],
+			additionalProperties: true,
+		},
+		agents: {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					agentId: {
+						type: 'string',
+					},
+				},
```

Comment:
This has a couple of more props namely `count` & `order`. Added them over here -> https://github.com/RocketChat/Rocket.Chat/pull/26410